block-storage-attacher/tests/e2e/README.md

```diff
@@ -99,7 +99,7 @@
 10. Now, run the following commands to perform the e2e test execution.
 	
 	```
-	cd $GOPATH/src/github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e
+	cd $GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e
```

Comment:
This should be `src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher`.
Check the above new path and change accordingly.